inference-engine/src/vpu/graph_transformer/src/stages/gather_elements.cpp

```diff
@@ -95,6 +114,23 @@ Stage StageBuilder::addGatherElementsStage(const Model &model,
         layer->name, StageType::GatherElements, layer, {input, indices}, {output});
 
     stage->attrs().set<int32_t>("axis", axis);
+    stage->attrs().set<int32_t>("rowIndicesMode", false);
+
+    return stage;
+}
+
+Stage StageBuilder::addExpGatherElementsStage(const Model &model,
+                                              const std::string &name,
+                                              const ie::CNNLayerPtr &layer,
+                                              const Data &input, const Data &indices,
+                                              const Data &rowIndices,
+                                              const Data &output,
+                                              int32_t axis) {
+    auto stage = model->addNewStage<GatherElementsStage>(
+        layer->name, StageType::GatherElements, layer, {input, indices, rowIndices}, {output});
+
+    stage->attrs().set<int32_t>("axis", axis);
+    stage->attrs().set<int32_t>("rowIndicesMode", true);
```

Comment:
I think you don't need `addExpGatherElementsStage`, you may pass the inputs as vector there, and depending on its size, set the attribute to `true` or `false`.

Reply:
Fixed